Explorer/Assets/Scripts/SceneRunner/Scene/SceneAssetBundleManifest.cs

```diff
@@ -59,6 +59,8 @@ public unsafe Hash128 ComputeHash(string hash)
         public bool Contains(string hash) =>
             ignoreConvertedFiles || convertedFiles.Contains(hash);
 
+        public bool TryGet(string hash, out string convertedFile) => convertedFiles.TryGetValue(hash, out convertedFile);
```

Comment:
You should supply a case-insensitive comparison for `convertedFiles = new HashSet<string>();`, otherwise the original logic is altered

Reply:
Ah I see, there is another `ctor` where it's defined without `StringComparer.OrdinalIgnoreCase`: confusion was mine 😄